src/server/frontend_xwayland/xwayland_surface.cpp

```diff
@@ -643,6 +640,28 @@ void mf::XWaylandSurface::move_resize(uint32_t detail)
     }
 }
 
+void mf::XWaylandSurface::WindowState::apply_change(
+    std::shared_ptr<XCBConnection> const& connection,
+    NetWmStateAction action,
+    xcb_atom_t net_wm_state)
+{
+    bool nil{false}, *prop_ptr = &nil;
```

Comment:
I guess it is pre-existing as the code has simply moved, but this isn't the project style.